src/platforms/android/configuration/integrations/apollo3.mdx

```diff
@@ -118,3 +119,93 @@ val apollo = ApolloClient.builder()
     }
     .build()
 ```
+
+## GraphQL Client Errors
+
+This feature, once enabled, automatically captures GraphQL client errors, like bad operations and response codes, as error events and reports them to Sentry. The error event will contain the `request` and `response` data, such as `url`, `status_code`, `data` (stringified `query`) and so on.
+
+Sentry will group GraphQL client errors by the `operationName`, `operationType` and `statusCode`, so that you can easily see how many errors happened for a specific `operationName`, `operationType` and `statusCode`.
+
+This feature is opt-in and can be enabled by setting the `captureFailedRequests` option to `true`:
+
+```kotlin
+import com.apollographql.apollo3.ApolloClient
+import io.sentry.apollo3.sentryTracing
+
+val apollo = ApolloClient.builder()
+    .serverUrl("https://your-api-host/")
+    .sentryTracing(captureFailedRequests = true)
+    .build()
+```
+
+By default, only GraphQL client errors with a response that includes the [errors](https://spec.graphql.org/October2021/#sec-Errors) array are captured as error events.
+
+GraphQL client errors from every target (`.*` regular expression) are automatically captured, but you can change this behavior by setting the `failedRequestTargets` option with either a regular expression or a plain `String`. A plain string must contain at least one of the items from the list. Plain strings don't have to be full matches, meaning the URL of a request is matched when it contains a string provided through the option.
+
```

Comment:
```suggestion
By default, only GraphQL client errors with a response that includes the [errors](https://spec.graphql.org/October2021/#sec-Errors) array will be captured as error events.

GraphQL client errors from every target (`.*` regular expression) will be automatically captured, but you can change this behavior by setting the `failedRequestTargets` option with either a regular expression or a plain `String`. Plain strings don't have to be full matches – the URL of a request will be matched if it contains a string provided through the option, but a plain string must contain at least one of the items from the list. 

```
From the list of errors? Can we specify the list of what please?

Reply:
I'm applying the fix for the 1st part, let me know if the 2nd part needs adaption after clarification here https://github.com/getsentry/sentry-docs/pull/7135/files#r1229248641